test/test_basic.py

```diff
@@ -95,9 +97,8 @@ def init_sync_from_geopackage(mc, project_name, source_gpkg_path, ignored_tables
 
     dbsync_init(mc, from_gpkg=True)
 
-
-def test_init_from_gpkg(mc):
-    project_name = 'test_init'
+@pytest.mark.parametrize("project_name", ['test_init', 'Test_Init'])
```

Comment:
this kind of parametrization may cause some issues on windows where the directory names are not case sensitive - maybe choose slightly different names? (e.g. `test_init_Uppercase` - to make it clear what is it about)